neuralmagic/benchmarks/scripts/backend_request_func.py

```diff
@@ -71,12 +77,12 @@ async def async_request_tgi(
         try:
             async with session.post(url=api_url, json=payload) as response:
                 if response.status == 200:
-                    data = ""
+                    data = None
                     async for part_data in response.content.iter_any():
                         if ttft == 0:
                             ttft = time.perf_counter() - st
                             output.ttft = ttft
-                        data += part_data
+                        data = part_data
```

Comment:
this doesn't not seem right as you will overwrite with part_data each time

Reply:
yeah - the thing is, part_data always has all the previous outputs - I verified it by printing out `part_data`. 

Also note that this script is copied from vllm upstream and getting the last `part_data` was the original functionality (https://github.com/vllm-project/vllm/blob/57f044945f25d90d1b434014b2719ba6b06fdc44/benchmarks/backend_request_func.py#L61)
I added the `data += part_data` and introduced a bug - this is basically a reversion.